src/main/java/mate/jdbc/service/CarService.java

```diff
@@ -0,0 +1,24 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+public interface CarService {
+    Car create(Car car);
+
+    Optional<Car> get(Long id);
```

Comment:
this method shouldn't return an Optional according to the task's description, fix this